test/functorch/test_aotdispatch.py

```diff
@@ -1,4 +1,5 @@
 # Owner(s): ["oncall: pt2"]
+# ruff: noqa: F841
```

Comment:
I guess you fixed the file so this one is not needed?

Reply:
Oops, gone.